kaspad/src/args.rs

```diff
@@ -176,6 +178,7 @@ pub fn cli() -> Command {
     let cmd = Command::new("kaspad")
         .about(format!("{} (rusty-kaspa) v{}", env!("CARGO_PKG_DESCRIPTION"), version()))
         .version(env!("CARGO_PKG_VERSION"))
+        .arg(arg!(-C --configfile <CONFIG_FILE> "Path of config file."))
```

Comment:
yes, now that I've tested it more thoroughly it appears that defaults take precedence if exist and clap's `get_one()` always return `Some(default)` so `unwrap_or()` didn't work like I'd expect.

I made a kinda hacky fix for it, it's possible that a refactor for the `Args` struct so that it's auto parsed using clap's `Parser` derive macro can be a prettier solution but I didn't want make so many changes.